coda-automation/scripts/release-docker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Also seems like CRUFT?? Thinking it's been deprecated for per services Makefiles (e.g. https://github.com/MinaProtocol/coda-automation/blob/master/services/coda-network-services/Makefile)